src/core/types/i18n.js

```diff
@@ -0,0 +1,4 @@
+/* @flow */
+
+// TODO: create a real type for i18n
```

Comment:
Haha. Up to this point while reading the PR I was excited that you had defined it 😛 It's not too important though. Good call on making a placeholder.